specifyweb/workbench/views.py

```diff
@@ -477,7 +477,9 @@ def dataset(request, ds: models.Spdataset) -> http.HttpResponse:
                     try:
                         validate(plan, upload_plan_schema.schema)
                     except ValidationError as e:
-                        return http.HttpResponse(f"upload plan is invalid: {e}", status=400)
+                        # TODO fix this
+                        # return http.HttpResponse(f"upload plan is invalid: {e}", status=400)
+                        pass
```

Comment:
Not optional! 

This should be fixed. 
If the upload plan is invalid, the upload plan will be assumed to be valid, and proceed with parsing the invalid plan and setting it on the Dataset.